packages/cli/src/lib/defaults/infra-modules/eth-ens-ipfs/docker-compose.yaml

```diff
@@ -1,6 +1,7 @@
 version: '3'
 services:
   ganache:
+    platform: linux/x86_64
```

Comment:
Please drop this, as `ganache` doesn’t require this by default, while it causes trouble inside WSL.
For more details see: https://github.com/polywrap/toolchain/pull/1581

Reply:
Yes, 100%! Adding it back in was a mistake. I didn't mean to commit it.

I was using it because I am actually having issues without it: https://github.com/polywrap/toolchain/pull/1581#issuecomment-1446268122